server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -783,6 +783,14 @@ public static IndexMergePolicy fromString(String text) {
         Property.IndexScope
     );
 
+    public static final Setting<Integer> INDEX_QUERY_CACHE_MIN_FREQUENCY = Setting.intSetting(
+        "index.query_cache.min_frequency",
+        5,
+        1,
+        Property.Dynamic,
+        Property.IndexScope
```

Comment:
Is there any reason, why we are limiting this to index level? As otherwise, it will require one to fine tune specific indices which may be hard, considering query cache doesn't give insights as to which index queries are getting cached.

Reply:
`TrackingQueryCachingPolicy ` is created every shard, so I make it as index level. It seems simpler to be the node level, I will changed it to be node level.